pkg/validation/spec/paths.go

```diff
@@ -114,7 +114,9 @@ func (p *Paths) UnmarshalNextJSON(opts jsonv2.UnmarshalOptions, dec *jsonv2.Deco
 				p.Paths[k] = pi
 			default:
 				_, err := dec.ReadValue() // skip value
-				return err
+				if err != nil {
+					return err
+				}
```

Comment:
should this merge first for backporting? also, should there be a testcase to exercise this?